frontend/components/ResourceInfoCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button, Grid, Paper, Sx, Text, rem } from '@mantine/core';
+import Link from 'next/link';
+import React from 'react';
+import { ResourceInfo } from '@/util/types/fhir';
+import { ExternalLink } from 'tabler-icons-react';
+
+export interface ResourceInfoCardProps {
+  resourceInfo: ResourceInfo;
+}
+
+export default function ResourceInfoCard({ resourceInfo }: ResourceInfoCardProps) {
+  return (
+    <Paper
+      shadow="sm"
+      p="md"
+      sx={theme => {
+        const style: Sx = {
+          ':hover': {
+            cursor: 'pointer',
+            backgroundColor: theme.colors.gray[0]
+          },
+          borderRadius: 6,
+          border: `${rem(1)} solid ${theme.colors.gray[3]}`
+        };
+        return style;
+      }}
+    >
+      <Grid align="center">
+        <Grid.Col span={10}>
+          <div>
+            <Text size="lg" fw={700}>
+              {resourceInfo.name ? resourceInfo.name : `${resourceInfo.resourceType}/${resourceInfo.id}`}
+            </Text>
+          </div>
+          {resourceInfo.version ? (
+            <Text size="sm">
+              {`v${resourceInfo.version}`} {resourceInfo.status && <b> ({resourceInfo.status})</b>}
+            </Text>
+          ) : (
+            <Text size="sm">{resourceInfo.status && <b>Status: {resourceInfo.status}</b>}</Text>
+          )}
+          {resourceInfo.url && (
+            <Text size="sm">
+              {resourceInfo.url}
+              {resourceInfo.version && `|${resourceInfo.version}`}
+            </Text>
+          )}
+          {resourceInfo.identifier && <Text size="sm">{`Identifier: ${resourceInfo.identifier}`}</Text>}
```

Comment:
Love the new layout! I think the Identifier line is the only one that still looks a little off. I wonder if even just italicizing the word `Identifier` would be an improvement, or doing something more to signify that it's not part of the value

Reply:
I like the idea of italicizing `Identifier`. Here's what it looks like:
![Screenshot 2023-05-09 at 9 01 27 AM](https://github.com/projecttacoma/measure-repository/assets/87094479/c01b3035-369d-45d9-b035-03728ea65195)

Let me know what you think - having trouble figuring out the best way to display this (or if we should even display it in the first place). I think this might be good for now and we can make adjustments later depending on the artifacts that we end up creating